frontend/src/pages/misc/terms-and-conditions.tsx

```diff
@@ -0,0 +1,114 @@
+
+        <h2 className="text-lg md:text-xl font-semibold">For European Union (EU) Users</h2>
+        <p className="mb-4 md:mb-6">If You are a European Union consumer, you will benefit from any mandatory provisions of the law of the country in which you are resident in.</p>
+
+        <h2 className="text-lg md:text-xl font-semibold">United States Legal Compliance</h2>
+        <p className="mb-4 md:mb-6">You represent and warrant that (i) You are not located in a country that is subject to the United States government embargo, or that has been designated by the United States government as a &quot;terrorist supporting&quot; country, and (ii) You are not listed on any United States government list of prohibited or restricted parties.</p>
+
+        <h2 className="text-lg md:text-xl font-semibold">Severability and Waiver</h2>
+        <h3 className="font-semibold">Severability</h3>
+        <p className="mb-4 md:mb-6">If any provision of these Terms is held to be unenforceable or invalid, such provision will be changed and interpreted to accomplish the objectives of such provision to the greatest extent possible under applicable law and the remaining provisions will continue in full force and effect.</p>
+        <h3 className="font-semibold">Waiver</h3>
+        <p className="mb-4 md:mb-6">Except as provided herein, the failure to exercise a right or to require performance of an obligation under these Terms shall not effect a party&apos;s ability to exercise such right or require such performance at any time thereafter nor shall the waiver of a breach constitute a waiver of any subsequent breach.</p>
+
+        <h2 className="text-lg md:text-xl font-semibold">Translation Interpretation</h2>
+        <p className="mb-4 md:mb-6">These Terms and Conditions may have been translated if We have made them available to You on our Service.
+            You agree that the original English text shall prevail in the case of a dispute.</p>
+
+        <h2 className="text-lg md:text-xl font-semibold">Changes to These Terms and Conditions</h2>
+        <p>We reserve the right, at Our sole discretion, to modify or replace these Terms at any time. If a revision is material We will make reasonable efforts to provide at least 30 days notice prior to any new terms taking effect. What constitutes a material change will be determined at Our sole discretion.</p>
+        <p className="mb-4 md:mb-6">By continuing to access or use Our Service after those revisions become effective, You agree to be bound by the revised terms. If You do not agree to the new terms, in whole or in part, please stop using the website and the Service.</p>
+
+        <h2 className="text-lg md:text-xl font-semibold" id="contact">Contact Us</h2>
+        <p>If you have any questions about these Terms and Conditions, You can contact us by email:</p>
```

Comment:
```suggestion
        <p>If you have any questions about these Terms and Conditions, You can contact us:</p>
```

Do not mention method of contact